build.ps1

```diff
@@ -774,6 +774,7 @@ function Build-Runtime($Arch)
         SWIFT_PATH_TO_LIBDISPATCH_SOURCE = "$SourceCache\swift-corelibs-libdispatch";
         SWIFT_PATH_TO_STRING_PROCESSING_SOURCE = "$SourceCache\swift-experimental-string-processing";
         SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE = "$SourceCache\swift-syntax";
+        CMAKE_SHARED_LINKER_FLAGS = "/DEBUG /INCREMENTAL:NO /OPT:REF";
```

Comment:
I think we should also enable ICF?

Reply:
I think the ICF flag caused an error somewhere during the build. I'll get the error message.

Reply:
Nope. This doesn't happen anymore. Maybe an incremental build glitch.

Enabled ICF.